.eslintrc

```diff
@@ -4,7 +4,7 @@
 	"extends": ["eslint:recommended", "plugin:node/recommended"],
 	"env": {
 		"node": true,
-		"es6": true
+    "es6": true
```

Comment:
The GitHub spacing here is off. I double checked on my editor and it is spaced the same as `node`.